### PR TITLE
Announcements

### PR DESCRIPTION
**Pull request by zootos**
Adds a simple announcement system to the website. Allows for creation of three different types of announcements (INFO: blue, WARNING: yellow, ERROR: red), that can be set to show on either just the front page or on all pages. Support multiple announcements being shown at the same time, as well as having support for timed and indefinite announcements.

### CODE REVIEW

**Comment by zootos** [comment]
I had a bit of trouble deciding on the names to use for the different types of announcements. I am open for changes to the current names.

File: announcements/templates/announcements/create_announcement.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <form class="ui form" method="POST" action="">
+            {% csrf_token %}
+            <div class="ui two fields">
+                <div class="ui field">
+                    <label>{{ form.classification.label }}</label>
+                    {{ form.classification }}
+                </div>
+                <div class="ui field">
+                    <label>{{ form.link.label }}
+                        <span data-content="{% trans "An optional link to an information page" %}">
```
Comment (hermabe) [inline announcements/templates/announcements/create_announcement.html]: Maybe use the `help_text` Django field attribute for these?

File: locale/nb/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1976,42 +2058,37 @@ msgstr "Se gammelt bilde"
...
-
 #: web/widgets.py:16 web/widgets.py:26
 msgid "Choose value"
 msgstr "Velg verdi"
 
 #: web/widgets.py:51
 msgid "Search places"
 msgstr "Søk etter steder"
+
+#~ msgid "Error"
```
Comment (hermabe) [inline locale/nb/LC_MESSAGES/django.po]: These are not used

**Comment by zootos** [comment]
> Looks good. Just a couple small comments. Also, some translations were missing, but was fixed by running `compilemessages`.

I was changing the translations around in the end, and guess I missed a compilemessages after the last change or two.

> "Info" and "Warning" names are good. "Error" is kind of misleading as there are no errors. Maybe something like "Critical", I'm not sure.

Critical is better. I will change to it for now, and then maybe somebody has a better suggestion

File: announcements/templates/announcements/create_announcement.html (hunk 1 above)
Comment (zootos) [reply announcements/templates/announcements/create_announcement.html]: Absolutely! I have never seen that field before, TIL.

**Pull request merged**